libraries/ui-library/src/components/six-input/six-input.tsx

```diff
@@ -96,40 +96,40 @@ export class SixInput {
   @Prop({ reflect: true }) readonly = false;
 
   /** The minimum length of input that will be considered valid. */
-  @Prop({ reflect: true }) minlength: number;
+  @Prop({ reflect: true }) minlength?: number;
 
   /** The maximum length of input that will be considered valid. */
-  @Prop({ reflect: true }) maxlength: number;
+  @Prop({ reflect: true }) maxlength?: number;
 
   /** The input's minimum value. */
-  @Prop({ reflect: true }) min: number;
+  @Prop({ reflect: true }) min?: number;
 
   /** The input's maximum value. */
-  @Prop({ reflect: true }) max: number;
+  @Prop({ reflect: true }) max?: number;
 
   /** The input's step attribute. */
-  @Prop({ reflect: true }) step: number;
+  @Prop({ reflect: true }) step?: number;
 
   /** A pattern to validate input against. */
-  @Prop({ reflect: true }) pattern: string;
+  @Prop({ reflect: true }) pattern?: string;
 
   /** Set to true to make the input a required field. */
-  @Prop({ reflect: true }) required: boolean;
+  @Prop({ reflect: true }) required = false;
 
   /** The input's autocaptialize attribute. */
-  @Prop() autocapitalize: string;
+  @Prop() autocapitalize: string = 'off';
 
   /** The input's autocorrect attribute. */
-  @Prop() autocorrect: string;
+  @Prop() autocorrect: 'on' | 'off' = 'off';
 
   /** The input's autocomplete attribute. */
-  @Prop() autocomplete: string;
+  @Prop() autocomplete: string = 'off';
```

Comment:
'on' | 'off' ?

Reply:
There are [many more possible values](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) for autocomplete, so I opted for `string` here.